enjoy/src/renderer/components/medias/media-player-controls.tsx

```diff
@@ -174,8 +174,8 @@ export const MediaPlayerControls = (props: {
           >
             <GaugeIcon className="w-6 h-6" />
             {playbackRate != 1.0 && (
-              <span className="absolute left-[0.9rem] bottom-0 text-[0.70rem] text-white">
-                {playbackRate.toFixed(2)}
+              <span className="absolute left-[1rem] top-6 text-[0.65rem] text-red-500">
```

Comment:
`red` is too much. Maybe just remove `text-white`, leaving it as default.

Reply:
how about grey? I think grey is better than default black.
<img width="493" alt="image" src="https://github.com/xiaolai/everyone-can-use-english/assets/13919152/d16f51d2-9f38-41d8-864a-9fb66a45e1b3">